src/CommandingOfficers/Commander.java

```diff
@@ -5,10 +5,17 @@
 
 import Terrain.GameMap;
 import Terrain.Location;
+import Units.AAModel;
```

Comment:
I'd rather just spell out AntiAir. It's not that much longer, and is more clear.